### PR TITLE
Add blog post about forensic container analysis

### PR DESCRIPTION
**Pull request by adrianreber**
This adds a blog article how to use to do forensic analysis on a container checkpoint.

https://github.com/kubernetes/enhancements/issues/2008

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Some early feedback - hope it's useful

File: content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md
Comment (adrianreber) [reply content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md]: done

File: content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md
Comment (adrianreber) [reply content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md:106]: done

File: content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md
Comment (adrianreber) [reply content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md]: changed

File: content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md
Comment (adrianreber) [reply content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md]: added

File: content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md
Comment (adrianreber) [reply content/en/blog/_posts/2023-03-10-forensic-container-analysis/index.md:16]: also added a sentence concerning alpha state

**Comment by adrianreber** [comment]
> Some early feedback - hope it's useful

Thanks. As always, very helpful.

**Comment by adrianreber** [comment]
@haircommander @saschagrunert please take a look, it is not as CRI-O related as the previous blog post, but as you also reviewed the previous blog post, maybe you also have some input for this PR. Thanks.

**Comment by haircommander** [comment]
LGTM, interesting blog and seems useful!

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm
/approve

/hold
as there is feedback available.

If you want to @adrianreber you can open another PR that updates your previous article to link to this follow up.
(/hold that other PR until this one publishes)

**Comment by sftim** [comment]
@adrianreber you can unhold this PR right away, or you can do the wording tweak and get a new LGTM. Your choice.

**Comment by adrianreber** [comment]
> @adrianreber you can unhold this PR right away, or you can do the wording tweak and get a new LGTM. Your choice.

I added your suggestions and re-pushed.

**Comment by adrianreber** [comment]
> If you want to @adrianreber you can open another PR that updates your previous article to link to this follow up.
> (/hold that other PR until this one publishes)

#39842

**Review by saschagrunert — APPROVED** [review]
LGTM, great blog post!

**Review by sftim — COMMENTED** [review]
/lgtm
/hold cancel

**Pull request merged**
